.github/workflows/reusable-build-docs.yaml

```
@@ -68,6 +74,15 @@ jobs:
       run: echo VERSION="${PACKAGE_VERSION}" >> "${GITHUB_ENV}"
 
     - name: Build the Ansible community package docs
+      if: env.REDIRECTS == 'true'
+      run: >-
+        make webdocs EXTRA_TAGS="-t redirects" ANSIBLE_VERSION="${{
+          env.PACKAGE_VERSION != 'devel' && env.PACKAGE_VERSION || ''
+        }}"
+      working-directory: build-directory/docs/docsite
+
+    - name: Build the Ansible community package docs
+      if: env.REDIRECTS == 'false'
       run: >-
         make webdocs ANSIBLE_VERSION="${{
```

Maybe just make this bit conditional instead of having two same-named steps with most things copied twice?
```suggestion
        make webdocs ${{
          fromJSON(env.REDIRECTS) && '-t redirects' || ''
        }} ANSIBLE_VERSION="${{
```